.golangci.yml

```diff
@@ -102,7 +99,7 @@ linters-settings:
   goimports:
     # put imports beginning with prefix after 3rd-party packages;
     # it's a comma-separated list of prefixes
-    local-prefixes: github.com/dapr/dapr
+    local-prefixes:
```

Comment:
Do you need `goimports` even if you use no particular options?

Reply:
oh I leave this one as placeholder, so that we can use it as reference later